pulsar-sql/presto-pulsar/pom.xml

```diff
@@ -113,11 +113,22 @@
             <version>${javax.annotation-api.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-utils</artifactId>
+            <version>3.0.16</version>
```

Comment:
It's better to add the version to properties

Reply:
fixed